tcf_website/tests/test_utils.py

```diff
@@ -202,3 +290,12 @@ def new_function(*args, **kwargs):
         logger.setLevel(previous_logging_level)
 
     return new_function
+
+
+def create_new_sem(obj, year):
+    """Helper method to modify current semester"""
+    obj.semester = Semester.objects.create(
+        year=year,
+        season='FALL',
+        number=1508
```

Comment:
Just an FYI: this `number` is a concatenation of `1`, the last two digits of `year`, and the starting month of the `season`. This _might_ confuse developers if there is a test fixture that does not conform to this. Should be something along this line:
```suggestion
        number=f'1{year % 100}8'